tests/snmpsim/ray3.snmprec

```diff
@@ -1,7 +1,5 @@
 1.3.6.1.2.1.1.1.0|4|Microwave Link
 1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.33555.4.1
-1.3.6.1.2.1.1.3.0|67|190478100
-1.3.6.1.2.1.1.5.0|4|<private>
```

Comment:
did you remove this on purpose? we prefer to keep them

Reply:
Only the mempool is added, as can be seen, so nothing else has changed. If the oid is not read, Racom is to blame.

Reply:
I'll try to regenerate snmprec. Both are displayed to me in LibreNMS.